packages/core/src/NodeExecuteFunctions.ts

```diff
@@ -2041,7 +2041,7 @@ export function getNodeParameter(
 		);
 		cleanupParameterData(returnData);
 	} catch (e) {
-		if (e instanceof ExpressionError && node.continueOnFail && node.name === 'Set') {
+		if (e instanceof ExpressionError && node.continueOnFail && node.type === 'n8n-nodes-base.set') {
```

Comment:
this is the weirdest hack around.. can we add a comment to explain what's going on here? a link to a story to clean this up?

Reply:
also some tests for this issue?

Reply:
@mutdmour It's explained in the story.

Reply:
What should we add to better explain it? I don't have context on why this is here.

Reply:
okay then maybe just a link to the related story?